LTHPasscodeViewController/LTHPasscodeViewController.m

```diff
@@ -1197,6 +1197,12 @@ - (void)showLockScreenWithAnimation:(BOOL)animated withLogout:(BOOL)hasLogout an
 - (void)showLockScreenOver:(UIView *)superview withAnimation:(BOOL)animated withLogout:(BOOL)hasLogout andLogoutTitle:(NSString*)logoutTitle {
     [self _prepareAsLockScreen];
     
+    // Add nav bar & logout button if specified
+    if (hasLogout) {
+        _isUsingNavBar = hasLogout;
+        [self _setupNavBarWithLogoutTitle:logoutTitle];
```

Comment:
why the position of the block of code matters here?

Reply:
Cause here may return and never load the nav bar.
![image](https://user-images.githubusercontent.com/70303100/109093118-b3230980-777c-11eb-949e-fde696847680.png)

Reply:
ah, ok, tricky